src/listings.ts

```diff
@@ -21,52 +21,52 @@ const {
   createPrintPurchaseReceiptInstruction,
   createCancelInstruction,
   createCancelListingReceiptInstruction,
-} = instructions;
+} = instructions
 
 export interface PostListingParams {
-  amount: number;
-  nft: Nft;
+  amount: number
```

Comment:
why are you ditching the semicolons?

Reply:
Hey @austinmilt, I was actually using the .prettierrc from marketplace repo. Seems holaplex and marketplace repo follow slightly different conventions. I can update it to match it with that of holaplex, if you want me to.